test/e2e/kubectl/kubectl.go

```diff
@@ -847,6 +847,9 @@ metadata:
 
 			// Pod
 			forEachPod(func(pod v1.Pod) {
+				gte, err := framework.KubectlVersionGTE(utilversion.MustParseSemantic("v1.9.0-alpha.2"))
```

Comment:
Server version should be checked instead

Reply:
Why checking server version? The describe behavior is purely client-side, isn't?

Reply:
Responded here: https://github.com/kubernetes/kubernetes/pull/56508#issuecomment-348029287

Reply:
Actually, skip both server and client.